.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "ecmaFeatures": {
+    "jsx": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "quotes": 1,
+    "no-trailing-spaces": 1,
+    "strict": 1,
```

Comment:
I've been told on co-op that use strict is good, but I have never looked into it. Hashtag my two cents.